### PR TITLE
tools: route_graph - added README file

### PR DESCRIPTION
**Pull request by VoIPNuggets-com**
#### Pre-Submission Checklist

- [x] Commit message has the format required by CONTRIBUTING guide
- [x] Commits are split per component (core, individual modules, libs, utils, ...)
- [x] Each component has a single commit (if not, squash them into one commit)
- [x] No commits to README files for modules (changes must be done to docbook files
in `doc/` subfolder, the README file is autogenerated)

#### Type Of Change
- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)

#### Checklist:

- [ ] PR should be backported to stable branches
- [ ] Tested changes locally
- [ ] Related to issue #XXXX (replace XXXX with an open issue number)

### CODE REVIEW

File: misc/tools/route_graph/README [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+1. Clone the GitHub repository:
+    git clone https://github.com/kamailio/kamailio.git
+2. Navigate to the tool's directory:
+    cd kamailio/misc/tools/route_graph/
+3. Run the script on your Kamailio configuration file:
+    python3 route_graph.py /path/to/kamailio.cfg
+
+For a more detailed walkthrough, please refer to this article: 
+Visualizing Kamailio Configuration File Using Route Graph Tool.
+http://voipnuggets.com/2023/06/13/visualizing-kamailio-configuration-file-using-route_graph-tool/
```
Comment (oej) [inline misc/tools/route_graph/README:14]: Time to get HTTPS:// - certificates are free.

Comment (VoIPNuggets-com) [reply misc/tools/route_graph/README]: It is actually https. I have updated the URL.

**Comment by linuxmaniac** [comment]
can you please squash both commits and add ``[skip ci]`` in the commit message?

**Comment by linuxmaniac** [comment]
Thanks

**Pull request merged**